### ISSUE TITLE
analysis-v2 questions

### ISSUE THREAD

**Issue by ACEnglish** [TimD1/vcfdist#16]
I have a few starter questions about some scripts inside `analysis-v2`.

1) 
https://github.com/TimD1/vcfdist/blob/9838e17b053e27573b6b0d3cfb852cfc47ba899c/analysis-v2/vs_prior_work/1_eval_vcfs.sh#L75-L83

This line does not match up with the documentation of how truvari refine runs whole genome analysis ([details](https://github.com/ACEnglish/truvari/wiki/bench#refining-bench-output)].  Is there a reason for this? 

2)
https://github.com/TimD1/vcfdist/blob/9838e17b053e27573b6b0d3cfb852cfc47ba899c/analysis-v2/vs_prior_work/1_eval_vcfs_tr.sh#L30

This line appears to be using the very first prototype version of TR regions (v4.20). Would this command also work on the [v1.0 release](https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/release/AshkenazimTrio/HG002_NA24385_son/TandemRepeats_v1.0/GRCh38/)? Also, should there be any consideration for Tier1 vs Tier2 regions?

3)
https://github.com/TimD1/vcfdist/blob/9838e17b053e27573b6b0d3cfb852cfc47ba899c/analysis-v2/phab_cm/1_phab_eval_vcfs.sh#L6

Could the [changes to phab](https://github.com/acenglish/truvari/wiki/Updates) since v4.0 affect results?

**Comment by ACEnglish** [comment]
Can you  please clarify why you "wanted to use the original coordinates from the GIAB-Q100 benchmarking BED". Running truvari refine for whole genome analysis without `--use-region-coords` is equivalent to forcing vcfdist to use the Q100 bed regions to define its 'superclusters'.

**Comment by TimD1** [comment]
When evaluating Truvari, I decided to use the default behavior (i.e. not including the optional `--use-region-coords` parameter) whenever I did not fully understand the impact such a flag would have on the analysis. Based on the short description of [`--use-region-coords`](https://github.com/ACEnglish/truvari/wiki/refine#--use-region-coords) in the documentation, it sounded like if I wanted to use Truvari refine to evaluate the GIAB-Q100 benchmarking regions, I should omit the flag and Truvari refine would evaluate the same regions as Truvari bench.

Based on your comment, it appears that this choice caused Truvari refine to run much slower than it otherwise would? Would this have impacted the accuracy, or just the runtime of Truvari refine?
